### PR TITLE
PKX-246 Added user course enrollments API 

### PR DESCRIPTION
**Pull request by mubtadaali**

### CODE REVIEW

**Review by emzubair — CHANGES_REQUESTED** [review]
Can we add docstrings for classes and methods as it is necessary for `pylint`

**Comment by tasawernawaz** [comment]
> Can we add docstrings for classes and methods as it is necessary for `pylint`

maza

File: openedx/features/pakx/lms/pakx_admin_app/serializers.py [hunk 1]
```diff
@@ -3,10 +3,42 @@
...
+
+    @staticmethod
+    def get_enrollment_date(obj):
+        return obj.created.strftime('%Y-%m-%d')
+
+    def get_progress(self, obj):
+        return get_course_progress_percentage(self.context['request'], text_type(obj.course.id))
+
+    @staticmethod
+    def get_grades(obj):
```
Comment (emzubair) [inline openedx/features/pakx/lms/pakx_admin_app/serializers.py:37]: are we using this method?

**Review by emzubair — APPROVED** [review]
👍🏽

File: openedx/features/pakx/lms/pakx_admin_app/serializers.py (hunk 1 above)
Comment (mubtadaali) [reply openedx/features/pakx/lms/pakx_admin_app/serializers.py:37]: yes, we are

**Pull request merged**